### ISSUE TITLE
Identify project data and describe analysis goals

### ISSUE THREAD

**Issue by coulbyph** [coulbyph/project-coulbyph#1]
Hi! Looking forward to the next stage @larnsce @mianzg @sskorik01

**Comment by larnsce** [comment]
Thank you, @coulbyph. It’s all set up as required and your data and analysis goals sound exciting.

You can start working with your data and open a new issue on this repository if you have questions or face problems. Please tag us when you open the issue.

I will close this issue now.

**Comment by larnsce** [comment]
Thank you, @coulbyph. It’s all set up as required and your data and analysis goals sound exciting.

You have only missed one thing. 

1. The repository is setup on your personal profile and it would be helpful for us if you set it up in the GitHub organisation for the course. Could you please set this up again here: https://github.com/organizations/ds4owd-001/repositories/new

2. Let me know on the new repo that you are done by opening an issue on the new repo

You can then start working with your data and open a new issue on the new repository if you have questions or face problems. Please tag us when you open the issue.

**Comment by coulbyph** [comment]
Thanks for the comments @larnsce .
Should I delete the project created on my personal profile?

**Comment by larnsce** [comment]
Yes, but only once you have set up the new one.
